main.go

```diff
@@ -150,6 +159,10 @@ func main() {
 		fmt.Fprintln(os.Stderr, "[41;1mWarning: agent forwarding not enabled[K[m")
 	}
 
+	if os.Getenv("JUMP_IP") == "public" {
```

Comment:
I'm roughly neutral.

Would be it `1` (public) versus anything else (private); or set to anything (public) versus empty/not-set (private)?